TCPserver.cpp

```diff
@@ -0,0 +1,63 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#define _XOPEN_SOURCE_EXTENDED 1
+#include <unistd.h> //included this for the close function
+
+#include <netinet/in.h>
+
+int main()
+{
+    // data to be sent to cleints
+    char server_message[256] = "Recieved message FUCK";
+
+    int server_socket;
+
+    // make socket call to get socket file descriptor
+    server_socket = socket(AF_INET, SOCK_STREAM, 0);
+
+    // define the server address
+    struct sockaddr_in server_address;
+
+    // specify family (AF_INET)
+    server_address.sin_family = AF_INET;
+
+    // specify port
+    server_address.sin_port = htons(9002);
```

Comment:
Make this configurable

Reply:
//using second string in command line as port number
int port_number = atoi(argv[1]);
...
server_addrerss.sin_port = htons(port_number);